pkg/storj/encryption.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package storj
+
+// EncryptionScheme is the scheme and parameters used for encryption
+type EncryptionScheme struct {
+	Algorithm      EncryptionAlgorithm
+	EncryptedKey   EncryptedPrivateKey
+	EncryptedNonce EncryptedNonce
```

Comment:
Is this the encrypted starting nonce for the segment?

If I am not wrong currently we don't keep track of it as we've decided to set the starting nonce to the segment's index. This it would be easier to implement multipart upload, while we still make sure the (key, nonce) pair is not reused and the satellite cannot reorder segments.

Reply:
Yes, ok, I'll remove this.